src/components/Header.jsx

```diff
@@ -9,24 +9,37 @@ import { useState, useEffect } from 'react'
 import { MoonIcon, SunIcon } from '@heroicons/react/outline'
 
 export function Header() {
-  const { systemTheme, theme = 'dark', setTheme } = useTheme()
+  const { systemTheme, theme = 'light', setTheme } = useTheme()
```

Comment:
One more thing. I've noticed a bug where when you first load the page, the logos/images will be the wrong theme. That goes away by removing this.
```suggestion
  const { systemTheme, theme, setTheme } = useTheme()
```

If you want to set a default theme, consider using the <ThemeProvider> in _app.js